src/todoList.viewModel.js

```diff
@@ -3,10 +3,20 @@ import {Todo} from './todo.model.js';
 
 export class ViewModel {
     constructor() {
-        this.todoList = [new Todo("Try javaScript"), new Todo("Buy a Unicorn")]        
+        this.todoList = ko.observableArray([]);        
```

Comment:
No need to specify [] as a parameter. Will be initialized to empty array by default